src/pages/MyCakesList.js

```diff
@@ -0,0 +1,52 @@
+import "./MyCakesList.css";
+import { Link } from "react-router-dom";
+import { RiFilePaper2Line } from "react-icons/ri";
+import ChCakeImg from "../images/chocolate.png";
+
+export default function MyCakesList() {
+  const recipes = JSON.parse(localStorage.getItem("recipesArray"));
+
+  function renderCakes() {
+    return recipes.map((cake, index) => {
+      const id = index + 1;
```

Comment:
I think at some point uuid will become handier for managing your recipes ids.

Reply:
you are right. uuid I am using when storing the input to localStorage. My idea was to have a clean count on pages ( /cakes/:id)